app/controllers/finders_controller.rb

```diff
@@ -8,6 +9,11 @@ class FindersController < ApplicationController
   def show
     slimmer_template "gem_layout_full_width" if i_am_a_topic_page_finder
 
+    if page_under_test?
+      @requested_variant = elastic_search_aa_test.requested_variant(request.headers)
+      @requested_variant.configure_response(response)
+    end
+
```

Comment:
I think all of this can be moved to the concern too, to keep the controller tidy.

Reply:
done!